lib/project-json-files.js

```diff
@@ -176,7 +176,7 @@ async function readFromPackagesWebsite(
   packageVersion,
   file
 ) {
-  // TODO [engine:node@>=18]: We can use `fetch` now.
+  // TODO(@lishaduck) [engine:node@>=21]: We can use `fetch` now.
```

Comment:
Fetch is available in 18, no?

Reply:
I thought so, but apparently it was only experimental until 21.